lib/presentation/screen/room_details/widgets/owner_section.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
+
+import '../../../design_system/theme/app_theme.dart';
+
+class RoomOwnerSection extends StatelessWidget {
+  final String name;
+  final String role;
+  final String imagePath;
+
+  const RoomOwnerSection({
+    super.key,
+    required this.name,
+    required this.role,
+    required this.imagePath,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = AppTheme.of(context);
+    return
+      ListTile(
+      leading: CircleAvatar(
+        backgroundImage: Image.asset(imagePath).image,
```

Comment:
```suggestion
backgroundImage: AssetImage(imagePath)
```
we need just pass object so it's enough to use AssetsImage